data/json/recipes/recipe_food.json

```diff
@@ -4892,13 +4892,14 @@
     "subcategory": "CSC_FOOD_OTHER",
     "skill_used": "cooking",
     "difficulty": 2,
-    "time": "4 m",
+    "time": "2 m 30 s",
     "autolearn": true,
     "batch_time_factors": [ 80, 2 ],
+    "charges": 1,
     "qualities": [ { "id": "COOK", "level": 2 } ],
-    "tools": [ [ [ "surface_heat", 7, "LIST" ] ] ],
+    "tools": [ [ [ "surface_heat", 4, "LIST" ] ] ],
     "//": "6u for the egg, 1u for the oil",
-    "components": [ [ [ "eggs_bird", 2, "LIST" ], [ "egg_reptile", 2 ] ], [ [ "any_butter_or_oil", 2, "LIST" ] ] ]
```

Comment:
this seems completly out of scope for this PR, if you want to change requirement for a recipe do it in a different PR with the accompanying justifications

Reply:
The goal of this PR is to give us the freedom to use any type of egg.
But the old recipe called for two identical eggs, so I changed it.